Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,22 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+
+  bottle :unneeded
+
+  depends_on "php" => :test
```

Comment:
How about making this `depends_on "php@7.2"` ? It will break when 7.3 or 8.0 is released.

Reply:
Fixed.

Reply:
Seems it cannot depend on an alias. Reverting back to PHP. In the event that PHP 7.3 or 8.0 is released these dependencies can also be updated. The same can be said if PHP 7.1 is ever dropped as well. It's a win-lose situation. You need to sacrifice somewhere.

https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/22319/version=el_capitan/testReport/brew-test-bot/el_capitan/audit_brew_php_switcher___online___new_formula/